design/accepted/gc_plus/gc-plus-cloud.md

```diff
@@ -0,0 +1,52 @@
+# Managed Uncommitted Garbage Collection for lakeFS cloud - Execution Plan
+
+## Goals
+1. Run UGC for lakeFS cloud users
+2. Allow Cloud users to clean up storage space by deleting objects from the underlying storage for cost efficiency
+3. Support new and existing cloud users (both repository structures)
+
+## Non-Goals
+1. Remove lakeFS limitations during the UGC job run
+2. Improve UGC OSS support
+
+### User Story
+When creating a new cloud installation - on the GC initializations page,
+add UGC initialization (default run job=true for all repositories, default sweep = true).
```

Comment:
Putting constraints aside, why would someone wouldn't want to run UGC on the repo? There are only upsides to it.

Reply:
Do you think we should run the UGC as default without the option for the user to disable it?